eng/Versions.props

```diff
@@ -1,9 +1,9 @@
 <Project>
   <PropertyGroup Label="Version settings">
     <!-- MSTest version -->
-    <VersionPrefix>3.7.0</VersionPrefix>
+    <VersionPrefix>3.8.0</VersionPrefix>
```

Comment:
this will go away before merge, but I need it for local development, and I expect some changes will be needed.